src/index.tsx

```diff
@@ -78,7 +78,8 @@ errorTracker.init();
 // so we need to explicitly set them
 const linkOptions = {
   credentials: "include",
-  uri: API_URI
+  uri: API_URI,
+  type: "image/png"
```

Comment:
Files can have many content types, we shouldn't hardcore them

Reply:
Reasoning behind the proposed solution:
 - based on the ticket description it’s about images
 - there is no generic image media-type and Content-Type can accept only single media-type   
 - I tested it with other image extensions and it looked good
 - png file extension is in the server response example in the ticket, so the solution seemed    
   reasonable for me also based on that